docs/src/appendix/snforge-library/declare.md

```diff
@@ -2,6 +2,6 @@
 
 > `fn declare(contract: ByteArray) -> Result<ContractClass, Array<felt252>>`
 
-Declares a contract for later deployment.
+Declares a contract for later deployment. The `contract` is the name of the contract (part after `mod` keyword) e.g. `"HelloStarknet"`.
```

Comment:
This is not needed, it's documented in cairo

Reply:
Then why do we have this page at all? I think it should be there

Reply:
Discoverability, interface

Reply:
Imo it should be here, but up 2 u